openquake/libs/__init__.py

```diff
@@ -16,4 +16,4 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = '2.2.2'
+__version__ = '2.2.3'
```

Comment:
In this case we have a broken back-compatibility (removing a package) => version could be 3.0.0,
Why do you consider it a bugfix ?

Reply:
That's right

Reply:
What do you think about a `2.3.0`? I would keep `3.0` for a full distribution bump (i.e. when we'll update all the libs, probably with 3.6)